doc/thirdpartypackages/index.rst

```diff
@@ -284,3 +284,16 @@ existing table. See the `blume github repository
 
 .. image:: /_static/blume_table_example.png
 
+GUI tool kit
+*******
```

Comment:
Must be of the same length as the text above. This is what fails the doc building CI,.